tools/issue-labeler/src/IssueLabelerService/IssueLabelerService.csproj

```diff
@@ -17,6 +17,7 @@
     <PackageReference Include="Azure.Search.Documents" Version="$(AzureSearchDocumentsVersion)" />
     <PackageReference Include="Microsoft.Extensions.Configuration" Version="$(MicrosoftExtensionsConfigurationVersion)" />
     <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonVersion)" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.AzureAppConfiguration" Version="8.1.1" />
```

Comment:
Even though it's not shared, worth keeping all the versions in one place, I'd say.